learn-ocaml-client.opam

```diff
@@ -18,7 +18,6 @@ homepage: "https://github.com/ocaml-sf/learn-ocaml"
 bug-reports: "https://github.com/ocaml-sf/learn-ocaml/issues"
 dev-repo: "git+https://github.com/ocaml-sf/learn-ocaml"
 depends: [
-  "asak" {>= "0.4"}
```

Comment:
ah I think I needed it in the end because `learnocaml_data` contains a type that is defined there, so not re-adding it to the plain `.opam` file was probably a mistake ?
This dependency could well be avoided, but I'm not sure the gain would be worth the added code complexity (needing to split the datatype modules, etc.)

Reply:
Ah actually it's there, just got moved around the list